changelog/deprecate_dualcontext.dd

```diff
@@ -1,13 +1,9 @@
 Deprecate local templates that receive local symbols by alias
 
-Support for local and member templates which can be instantiated with local
-symbols was added to the language in D 2.087.0, however there are a number of
-issues with the feature including:
+Support for instantiating local and member templates with local
+symbols was implemented in DMD 2.087.0. However, the implementation
+was incompatible with GDC and LDC backends.
 
-- Only DMD has code-gen support for the feature.
-- There has been no support to implement this feature in either GDC or LDC.
-- There is still no language specification that describes the feature.
-
-Because of this, there has been a dead feature introduced to the language for
-over nine releases, so now it has been deprecated, and due to be fully reverted
-in a future release.
+In order to maintain feature parity among D implementations, this
+improvement has been deprecated, and may be removed from a future DMD
```

Comment:
"may be removed from a future DMD"
May!? What do you mean by may? It quite clear that @ibuclaw intended to remove this in the feature. Don't give them any false hopes here.

Reply:
@ibuclaw has proposed alternative fixes. Although there is a small chance that anyone is going to actually implement those, I think it is better to stay positive.

Reply:
>@ibuclaw has proposed alternative fixes.
 >Although there is a small chance that anyone is going to actually implement those, I think it is better to stay positive.
>
Just because you proposed something doesn't mean it going to happen. Let not give them false hope here.